server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/model/JmapRfc8621Configuration.scala

```diff
@@ -22,15 +22,26 @@ package org.apache.james.jmap.model
 import java.net.URL
 
 import org.apache.commons.configuration2.Configuration
+import org.apache.james.jmap.routes.UploadRoutes
+import org.apache.james.jmap.routes.UploadRoutes.Size
+import org.apache.james.util
+import org.apache.james.util.Size
+
 
 object JmapRfc8621Configuration {
   val LOCALHOST_URL_PREFIX: String = "http://localhost"
   private def from(urlPrefixString: String): JmapRfc8621Configuration = JmapRfc8621Configuration(urlPrefixString)
   var LOCALHOST_CONFIGURATION: JmapRfc8621Configuration = from(LOCALHOST_URL_PREFIX)
   val URL_PREFIX_PROPERTIES: String = "url.prefix"
 
-  def from(configuration: Configuration): JmapRfc8621Configuration =
+  val UPLOAD_LIMIT_PROPERTIES: String = "upload.max.mb.size"
+  val UPLOAD_LIMIT_30_MB: String = "30M"
+  var MAXSIZE_UPLOAD: Option[util.Size] = None
+
+  def from(configuration: Configuration): JmapRfc8621Configuration = {
+    MAXSIZE_UPLOAD = Some(Size.parse(configuration.getString(UPLOAD_LIMIT_PROPERTIES, UPLOAD_LIMIT_30_MB)))
```

Comment:
I do not see documentation for this new configuration option. Can you add it for old documentation and new documentation?

Reply:
?